lib/core/jdl_application_configuration_factory.js

```diff
@@ -17,12 +17,18 @@
  * limitations under the License.
  */
 
+const logger = require('../utils/objects/logger');
```

Comment:
I'd rather have this import after the others

Reply:
Thanks. Fixed that. :smile: